webhook_server/tests/test_runner_handler.py

```
@@ -1889,6 +1910,398 @@ async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, st
+                assert "Jenia Peimer <jpeimer@redhat.com>" in command
+                return (True, "success", "")
+            return (True, "", "")
+
+        with (
+            patch(
+                "webhook_server.libs.handlers.runner_handler.run_command",
+                new=AsyncMock(side_effect=run_command_side_effect),
+            ),
+            patch(
+                "webhook_server.libs.handlers.runner_handler.github_api_call",
+                new=AsyncMock(side_effect=lambda fn, *a, **kw: fn() if callable(fn) else fn),
```

Ruff (0.15.10)

[warning] Unused lambda argument: `a`

(ARG005)